src/libs/PolicyUtils.ts

```diff
@@ -107,7 +107,7 @@ function isExpensifyGuideTeam(email: string): boolean {
 /**
  * Checks if the current user is an admin of the policy.
  */
-const isPolicyAdmin = (policy: OnyxEntry<Policy>): boolean => policy?.role === CONST.POLICY.ROLE.ADMIN;
+const isPolicyAdmin = (policy?: OnyxEntry<Policy>): boolean => policy?.role === CONST.POLICY.ROLE.ADMIN;
```

Comment:
Revert. Pass `null` if policy is not available. (same for other util changes)

Reply:
Oh, I don't know that. Updated.